src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -3125,7 +3125,6 @@ class DropIndexStatementSegment(BaseSegment):
     match_grammar: Matchable = Sequence(
         "DROP",
         "INDEX",
-        Ref.keyword("CONCURRENTLY", optional=True),
```

Comment:
If this is the only ref to it, can you drop it from the dialect_ansi_keywords too?

Reply:
Oh yeah, I meant to do that.  Done!